Purchases/Public/Offering.swift

```diff
@@ -27,7 +27,7 @@ import Foundation
     @objc public let serverDescription: String
 
     /**
-     Array of `RCPackage` objects available for purchase.
+     Array of `Package` objects available for purchase.
```

Comment:
double backtick

Reply:
✅ , also added it for a few other places where it was `package`